### PR TITLE
Add remove and add functionalities

### PR DESCRIPTION
**Pull request by Koshao-sys**
### **The requirements listed here have been applied in this pull request**

- [ ] Use webpack to bundle JavaScript.
- [ ] Learn how to use proper ES6 syntax.
- [ ] Use ES6 modules to write modular JavaScript.
- [ ] Remove all hardcoded items from the tasks array.
- [ ] Create a new JavaScript file for the new functionality.
- [ ] Implement a function for adding a new task (add a new element to the array).
- [ ] Implement a function for deleting a task (remove an element from the array).
- [ ] Implement a function for editing task descriptions.
- [ ] By default new tasks should have the property completed set to false and the property index set to the
value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal
to 5).
- [ ] Deleting a task should update all remaining items' indexes, so they represent the current list order and
are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set
to 1)
- [ ] All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: modules/list.js [hunk 1]
```diff
@@ -1,20 +1,94 @@
...
+
+  deleteList = (index) => {
+    // eslint-disable-next-line eqeqeq
+    this.todo = this.todo.filter((obj) => obj.index != index);
+    this.todo.forEach((item, index) => {
+      item.index = index + 1;
+    });
+    localStorage.setItem('todo', JSON.stringify(this.todo));
+    this.createlist();
+  }
```
Comment (roshan-bajgain) [inline modules/list.js:24]: - [ ] Everything is good to add is working fine, and edit is also working fine but when I try to delete the item then It is not deleting. I would suggest you please fix this issue and also make sure that after deleting it is updating properly on local storage.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Koshao-sys, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- [ ] Good design ✔️ 

## Required Changes ♻️
- [ ] I cannot leave there so I am leaving it here. Please In your getting section you have only setup how to clone project. I would suggest you please write how user can install the project and run on the browser. `npm install` and `npm start`.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/list.js (hunk 1 above)
Comment (Koshao-sys) [reply modules/list.js:24]: @roshan-bajgain Please note that I have rechecked and the delete functionality is working just fine.

Comment (roshan-bajgain) [reply modules/list.js:24]: @Koshao-sys, I tried twice and sometimes It used to remove and sometimes when I click twice it was not working.

Comment (roshan-bajgain) [reply modules/list.js:24]: @Koshao-sys, You can ask for another review. Sorry, It's working fine.

File: README.md [hunk 2]
```diff
@@ -45,11 +45,11 @@ After you're finished please remove all the comments and instructions!
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
-  <!--- [Prerequisites](#prerequisites)
+  <!--- [Prerequisites](#prerequisites)-->
```
Comment (rustam-tolipov) [inline README.md:48]: - [ ] As the previous reviewer suggested kindly work on your delete button. If I click middle or left side of delete button it's not responding, till I click the right top corner of the button. Please look throught the video and fix the issue thank you!


https://user-images.githubusercontent.com/69011963/220871635-81a16b74-cf3e-4cd7-93fa-70b67002714d.mov

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @Koshao-sys, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
_Great description on readme_

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/list.js (hunk 1 above)
Comment (roshan-bajgain) [reply modules/list.js:24]: @Koshao-sys, I tried twice please check it, as it is not working

**Review by OLIPLICHE — APPROVED** [review]
Hi @Koshao-sys, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Apply the change, the App is working fine👏 👏
# Optional
- I would advise you to remove the live demo section from your Readme if you are not deploying.  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**